db/bigtable_eth1.go

```diff
@@ -2849,7 +2841,7 @@ func (bigtable *Bigtable) GetEth1ERC721ForAddress(prefix string, limit int64) ([
 	return data, indexes[len(indexes)-1], nil
 }
 
-func (bigtable *Bigtable) GetAddressErc721TableData(address string, pageToken string) (*types.DataTableResponse, error) {
+func (bigtable *Bigtable) GetAddressErc721TableData(address []byte, pageToken string) (*types.DataTableResponse, error) {
```

Comment:
**issue:** This `string` => `[]byte` breaks the already present use of `address` namely for `pageToken`. For that you need to change `%s` => `%x`.
Currently we can't find data because of wrong keys and so the Erc721 tab never shows up.

The issue also for `GetAddressErc1155TableData`.

You can use **Prater** address **0xb64a30399f7f6b0c154c2e7af0a3ec7b0a5b131a** to test this.

Reply:
VERY good catch, thank you. Also :+1: for including a test address.
=> Fixed.